Portfolio/index.html

```diff
@@ -13,18 +14,22 @@
     <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
     <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
     <nav>
+    <h2>Jahnavi</h2>
         <ul>
             <li>Home</li>

```

Comment:
The closing form tag '</form>' should be moved above the line break '<br/>' to close the form correctly.